lib/rules/max-attributes-per-line.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @fileoverview Define the number of attributes allows per line
+ * @author Filipa Lacerda
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+const utils = require('../utils')
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Define the number of attributes allows per line',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    fixable: 'code',
+    schema: [{
+      type: 'object',
+      properties: {
+        'firstline': {
+          type: 'number',
+          minimum: 0
+        },
+        'singleline': {
+          type: 'number',
+          minimum: 1
+        },
+        'multiline': {
+          type: 'number',
+          minimum: 1
+        }
+      }
+    }]
+  },
+
+  create: function (context) {
+    const configuration = context.options[0]
+    const multilineMaximum = configuration.multiline
+    const singlelinemMaximum = configuration.singleline
+    const canHaveFirstLine = configuration.firstline !== 0
+
+    utils.registerTemplateBodyVisitor(context, {
+      'VStartTag' (node) {
+        const numberOfAttributes = node.attributes.length
+
+        if (!numberOfAttributes) return
+
+        if (isSingleLine(node) && numberOfAttributes > singlelinemMaximum) {
+          context.report({
+            node,
+            loc: node.loc,
+            message: 'It has more than {{singlelinemMaximum}} attributes per line.',
```

Comment:
I think better if this rule reports the violated attributes instead of elements which have the violated attributes.
Because editors can indicate more proper locations with red lines.